test/bench/index.js

```diff
@@ -1,10 +1,23 @@
 // TODO - dummy benchmark - should replace with log related benchmark
-import {Bench} from '@probe.gl/bench';
+const arr = [1, 1, 1, 1];
 
-const suite = new Bench({
-  id: 'test'
-});
+const someFn = i => ((((i * 3 * 8) / 1200) * 0.002) / 40) * 0.2;
 
-suite.group('probe.gl: No self benchmarks available yet (yarn test runs bench tests)');
+let sumForEach = 0;
+let sumMap = 0;
+let sumFor = 0;
 
-suite.run();
+export default function addBenchmarks(suite, addReferenceBenchmarks) {
```

Comment:
If this file is not actually running benchmark then maybe `index.js` is not the best name?